### ISSUE TITLE
[Debian Buster] pkg/prelude/bootstrap.js:1178 failing

### ISSUE THREAD

**Issue by gna582** [opendistro-for-elasticsearch/perftop#29]
When running perf-top-1.0.0.0-LINUX on my debian buster installation I receive this error:
```
❯ ~/bin/perf-top-linux --dashboard ClusterOverview --endpoint testnode-qa001.server.lan:9600
pkg/prelude/bootstrap.js:1178
      throw error;
      ^

Error: File '/**/estop-release/node_modules/blessed/usr/xterm' was not included into executable at compilation stage. Please recompile adding it as asset or script.
    at error_ENOENT (pkg/prelude/bootstrap.js:422:17)
    at readFileFromSnapshot (pkg/prelude/bootstrap.js:650:29)
    at Object.fs.readFileSync (pkg/prelude/bootstrap.js:693:18)
    at Tput.readTerminfo (/snapshot/estop-release/node_modules/blessed/lib/tput.js:149:13)
    at Tput.compileTerminfo (/snapshot/estop-release/node_modules/blessed/lib/tput.js:616:28)
    at Tput.injectTerminfo (/snapshot/estop-release/node_modules/blessed/lib/tput.js:620:27)
    at Tput._useXtermInfo (/snapshot/estop-release/node_modules/blessed/lib/tput.js:111:15)
    at Tput.setup (/snapshot/estop-release/node_modules/blessed/lib/tput.js:85:10)
    at new Tput (/snapshot/estop-release/node_modules/blessed/lib/tput.js:60:10)
    at Program.setupTput (/snapshot/estop-release/node_modules/blessed/lib/program.js:210:26)
```
xterm exists but it is somewhere else:
```
❯ which xterm
/usr/bin/xterm
```

I have no experience debugging node.js so that's all I can give you.

**Comment by Hubbitus** [comment]
Please see https://github.com/opendistro-for-elasticsearch/perftop/issues/36
I can reproduce it also on version `perf-top-1.4.0.0-LINUX`

**Comment by nickgal** [comment]
I ran into this problem with Ubuntu 20.04 and `perf-top-1.6.0.0-LINUX.zip`. 

After a little digging I was able to able to find a couple possible solutions:
  - Setting the `TERM` env var to `xterm` (ex. `TERM=xterm ./perf-top-linux`) which matches the env var from the `amazon/opendistro-for-elasticsearch` docker image. Ubuntu seemed to default it to `xterm-256color`. 
  - Adding the files from blessed as a pkg asset by adding the following line to `package.json` as mentioned https://github.com/zeit/pkg/issues/530#issuecomment-505728610
    ```"pkg": { "assets": "node_modules/blessed/**/*" }```
